addons/bestja_frontend_ucw/static/src/css/custom.css

```diff
@@ -21,6 +21,50 @@ a:hover, a:focus{
     text-decoration: none;
 }
 
+.field-join-info h4{
+    font-size: 14px;
+    color: #737373;
+}
+
+.field-join-info span{
+    font-size: 13px;
+}
+
+.field-join-info span:hover:after{
+    content: "Jeżeli chcesz zostać koordynatorem w swojej organizacji, wypełnij formularz, aby w kolejnym kroku uzupełnić dane swojej organizacji.";
```

Comment:
Thank you for this comment and suggestion. Content has been placed in new container, which is also narrower than before in previous implementation and more responsive in my opinion, because it is displaying in center of register form container.